src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/integration/GetPrisonNames.kt

```diff
@@ -78,6 +80,101 @@ class GetPrisonNames : IntegrationTest() {
     }
   }
 
+  @Test
+  fun `should return prison active name id only`() {
+    // Given
+    val endPoint = "/prisons/names?name=WDI"
+
+    // When
+    val responseSpec = webTestClient.get().uri(endPoint)
+      .exchange()
+
+    // Then
+    responseSpec.expectStatus().isOk
+
+    val prisonNames = getPrisonNames(responseSpec.expectBody())
+    Assertions.assertThat(prisonNames).isNotEmpty
+    with(prisonNames[0]) {
+      Assertions.assertThat(prisonId).isEqualTo("WDI")
+      Assertions.assertThat(prisonName).isEqualTo("Wakefield (HMP)")
+    }
+  }
+
+  // Different combinations of 'active' and 'name' for positive scenarios'
+  @ParameterizedTest
+  @CsvSource(
+    "true, WDI",
+    "null, WDI",
+    "false, WOI",
```

Comment:
make it easier for the reader and add a comment explaining that WDI is an active prison and WOI isn't